npm/webpack-dev-server/src/helpers/angularHandler.ts

```diff
@@ -30,21 +33,7 @@ type AngularJson = {
 
 const dynamicImport = new Function('specifier', 'return import(specifier)')
 
-export async function angularHandler (devServerConfig: WebpackDevServerConfig): Promise<PresetHandlerResult> {
-  const webpackConfig = await getAngularCliWebpackConfig(devServerConfig)
-
-  return { frameworkConfig: webpackConfig, sourceWebpackModulesResult: sourceDefaultWebpackDependencies(devServerConfig) }
-}
-
-async function getAngularCliWebpackConfig (devServerConfig: WebpackDevServerConfig) {
-  const { projectRoot } = devServerConfig.cypressConfig
-
-  const {
-    generateBrowserWebpackConfigFromContext,
-    getCommonConfig,
-    getStylesConfig,
-  } = await getAngularCliModules(projectRoot)
-
+export async function getProjectConfig (projectRoot: string): Promise<Cypress.AngularDevServerProjectConfig> {
```

Comment:
Might it be possible to throw a config error in this function (if we can't resolve the config with the defaults) that has some info about configuring your Angular project using `devServer.options.projectConfig`?